### PR TITLE
[Chore](runtime-filter) unify interfaces of bloom filter and remove some unused code

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
unify interfaces of bloom filter and remove some unused code

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/exprs/bloom_filter_func.h [hunk 1]
```diff
@@ -412,21 +401,10 @@ struct StringFindOp : public BaseOp {
         }
     }
 
-    static bool find(const BloomFilterAdaptor& bloom_filter, const void* data) {
+    bool find_olap_engine(const BloomFilterAdaptor& bloom_filter, const void* data) const override {
         const auto* value = reinterpret_cast<const StringRef*>(data);
-        if (value == nullptr) {
```
Comment (yiguolei) [inline be/src/exprs/bloom_filter_func.h:417]: why remove  value == nullptr check?

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/exprs/bloom_filter_func.h (hunk 1 above)
Comment (BiteTheDDDDt) [reply be/src/exprs/bloom_filter_func.h:417]: this is useless check

**Review by yiguolei — DISMISSED** [review]
lgtm

**Review by yiguolei — APPROVED** [review]
lgtm

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request merged**